examples/gno.land/r/chaegnal/organizer/group/proposal_actions.gno

```diff
@@ -0,0 +1,155 @@
+package group
+
+import (
+	"errors"
+	"std"
+	"strconv"
+	"strings"
+	"time"
+
+	"gno.land/p/nt/commondao"
+	"gno.land/r/chaegnal/organizer"
+)
+
+const (
+	acceptMemberFunc = "acceptAwaitingMember"
+	refuseMemberFunc = "refuseAwaitingMember"
+	makeAdminFunc    = "makeAdmin"
+	removeAdminFunc  = "removeAdminRank"
+	removeMemberFunc = "removeMember"
+	setHangoutFunc   = "SetHangoutTimeAndPlace"
+)
+
+type callbackProposalDefinition struct {
+	GroupId      string
+	TitleValue   string
+	BodyValue    string
+	Callback     string
+	CallbackArgs []string
+}
+
+func (c *callbackProposalDefinition) Title() string {
+	return c.TitleValue
+}
+
+func (c *callbackProposalDefinition) Body() string {
+	return c.BodyValue
+}
+
+func (c *callbackProposalDefinition) VotingPeriod() time.Duration {
+	return 24 * time.Hour
+}
+
+func (c *callbackProposalDefinition) Tally(r commondao.ReadonlyVotingRecord, m commondao.MemberSet) (bool, error) {
+	yesCount := r.VoteCount(commondao.ChoiceYes)
+	noCount := r.VoteCount(commondao.ChoiceNo)
+	return yesCount > noCount, nil
+}
+
+// CreateProposals allows an admin to create a proposal
+// It sanitizes the arguments given
+func CreateProposals(cur realm, groupId, title, desc, callback, callbackArgsStr, logged string) {
+	if !isAdmin(groupId, std.PreviousRealm().Address()) {
+		panic("only admin can create proposals")
+	}
+
+	group := organizer.GetGroupFromId(groupId)
+
+	callbackArgsStr = strings.Trim(callbackArgsStr, "\"")
+	callbackArgs := strings.Split(callbackArgsStr, ",")
+
+	def := &callbackProposalDefinition{
+		GroupId:      groupId,
+		TitleValue:   title,
+		BodyValue:    desc,
+		Callback:     callback,
+		CallbackArgs: callbackArgs,
+	}
```

Comment:
The proposal definition could store the caller (logged) address instead of receiving it though an arguments string, that way you can make sure is the right one and would remove redundancy